core/src/main/java/hudson/PluginManager.java

```diff
@@ -783,7 +783,7 @@ public boolean accept(File dir, String name) {
                 }
             });
 
-            LOGGER.log(INFO, "Upgraded Jenkins from version {0} to version {1}. Loaded detached plugins (and dependencies): {2}",
+            LOGGER.log(FINEST, "Upgraded Jenkins from version {0} to version {1}. Loaded detached plugins (and dependencies): {2}",
```

Comment:
This an important debugging help and shouldn't be lower than the default visible INFO.